src/mjolnir/adminbuilder.cc

```diff
@@ -20,9 +21,9 @@ BOOST_GEOMETRY_REGISTER_POINT_2D(valhalla::midgard::PointLL,
                                  boost::geometry::cs::geographic<boost::geometry::degree>,
                                  first,
                                  second);
-using ring_t = boost::geometry::model::ring<valhalla::midgard::PointLL>;
-using polygon_t = boost::geometry::model::polygon<valhalla::midgard::PointLL>;
-using multipolygon_t = boost::geometry::model::multi_polygon<polygon_t>;
+using ring_template = boost::geometry::model::ring<valhalla::midgard::PointLL>;
+using polygon_template = boost::geometry::model::polygon<valhalla::midgard::PointLL>;
+using multipolygon_template = boost::geometry::model::multi_polygon<polygon_template>;
```

Comment:
why the name change though? `_t` stands for type btw

Reply:
Strange.  I was getting a conflict before.  I just did a clean and it was fine.

Reply:
I will revert.